_projects/2018/astropy/ccdproc-optimize.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Optimizing performance of `ccdproc`
+desc: The package [ccdproc](https://ccdproc.readthedocs.io) has a stable API but little effort has been put in to optimizing its speed or memory use. The project is to do both optimizations.
+# add a short one line description of your project
+requirements:
+# Student requirements:
+ - Familiar with the basics of astronomical image reduction
+difficulty: medium
+issues:
+# Related issues (if any)  to this project.
+ - https://github.com/astropy/astropy/issue/4444
+ - https://github.com/astropy/astropy/issue/5555
+mentors:
+# First person in contact; mentors may change before project starts.
+ - mwcraig
```

Comment:
since this project comes up as one of the first ones, we definitely need to list more mentors even if the actual assignments will happen during student selection. 
As a placeholder you can add me, I'm happy to potentially co-mentor the CI/asv aspects of the project.